common/models/templates/build.sh

```diff
@@ -16,7 +16,7 @@ builder_describe "Builds the predictive-text model template implementation modul
   "@/common/web/keyman-version" \
   "@/common/web/es-bundling" \
   "@/common/models/wordbreakers" \
-  "@/common/web/utils" \
+  "@/web/src/engine/common/utils" \
```

Comment:
Note: this is only going to be okay if we are also moving common/models/templates over, otherwise we are introducing a deep dep from common -> web.

Reply:
Also of note:  some of my current Web work is likely to have `developer/src/kmc-model` import from `common/models/templates`.

Reply:
From #11994:

https://github.com/keymanapp/keyman/blob/5011d7fed68f79e0368f42471d2c74bc7742556d/common/models/templates/src/trie-builder.ts pulls in the primary Trie-construction code currently found in `kmc-model`.  We'll likely want it for user-dictionary support.

Accordingly...

https://github.com/keymanapp/keyman/blob/5011d7fed68f79e0368f42471d2c74bc7742556d/developer/src/kmc-model/src/build-trie.ts#L1-L4

Reply:
In a similar vein, #11088 will want to affect the same section of code, adding better Trie compression strategies.  The compression part could be kept kmc-model only in theory, but the decompression part will certainly be needed within the live model - and personally, it makes the most sense if the two parts can be tested together in parallel.  My _current_ plan is to place the definition for both within `common/models/templates`.

Reply:
OK, that's a bad dependency chain we are setting up for ourselves there -- developer/src/kmc-model -> common/models/templates -> web/src/engine/common/utils.

I think we need to discuss this more before we continue moving things around, because we are treading on each others' toes a bit, and it's liable to become (more of) a mess if we are not careful!